### PR TITLE
button: Increase height to 38px

### PR DESCRIPTION
**Pull request by dimitrisnl**
https://github.com/balena-io-modules/rendition/issues/847

The previous implementation was relying on space units.

### CODE REVIEW

**Comment by dimitrisnl** [comment]
`npx run jest -- -u` doesn't produce any new snapshots, yet the docker build fails. That's a bit strange.

File: src/components/Button.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const overrideHoverEffect = css`
 const ButtonBase = styled(Button)`
 	font-family: ${props => props.theme.titleFont};
 	font-weight: ${props => normal(props)};
-	height: ${props => px(props.theme.space[4])};
+	height: 38px;
```
Comment (sradevski) [inline src/components/Button.tsx]: Wouldn't it be nice if we can configure this through theming? I am OK with fixed value for now as well, just a thought.

**Comment by sradevski** [comment]
@dimitrisnl @LucianBuzzo  I was thinking it might be better to set the button's `white-space: nowrap` in this PR, since it has fixed height anyway. Any objections to that?

**Comment by dimitrisnl** [comment]
I agree, it will help with these weird cases

File: src/components/Button.tsx (hunk 1 above)
Comment (dimitrisnl) [reply src/components/Button.tsx]:
```js
  button: {
      border: {
          width: string;
          radius: string;
      };
      primary: {};
      disabled: {
          opacity: number;
      };
      minWidth: string;
      maxWidth: string;
      padding: {
          vertical: string;
          horizontal: string;
      };
  };
```

Grommet doesn't give the option in the theme.

Comment (sradevski) [reply src/components/Button.tsx]: Yeah, I meant we can extend the theme definition, but ignore it for now.

**Comment by dimitrisnl** [comment]
@balena-ci 	retest

File: src/components/Button.tsx (hunk 1 above)
Comment (LucianBuzzo) [reply src/components/Button.tsx]: @dimitrisnl @sradevski Lets fix this now, and extend the theme definition with a `height` value

**Comment by sradevski** [comment]
@dimitrisnl Maybe also include #822 in this PR, but as a separate commit.

File: src/components/Button.tsx (hunk 1 above)
Comment (sradevski) [reply src/components/Button.tsx]: @dimitrisnl can you please add `height` to the theme in this PR? Cheers

Comment (dimitrisnl) [reply src/components/Button.tsx]: Ah missed this. Will do 👍

**Pull request merged**